### PR TITLE
re-enable interleaved 1f1b test

### PR DESCRIPTION
**Pull request by H-Huang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* (to be filled)

### CODE REVIEW

File: test/test_pipeline_schedule.py [hunk 1]
```diff
@@ -320,38 +317,33 @@ def test_interleaved_1f1b(self):
...
+        ]
 
+        loss_fn = torch.nn.MSELoss()
         schedule = ScheduleInterleaved1F1B(
             stages,
             num_microbatches,
+            loss_fn=loss_fn,
         )
-        schedule.step_microbatches(microbatches)
+        schedule.step_microbatches(microbatches, target_mbs=target_mbs)
```
Comment (kwen2501) [inline test/test_pipeline_schedule.py:330]: Should we feed target to every rank?

File: test/test_pipeline_schedule.py [hunk 2]
```diff
@@ -320,38 +317,33 @@ def test_interleaved_1f1b(self):
...
         )
-        schedule.step_microbatches(microbatches)
-
-        # differing microbatch size
-        num_microbatches = 64
-        microbatches = [
-            torch.randn_like(microbatch) for _ in range(num_microbatches)
-        ]
-        schedule.step_microbatches(microbatches)
+        schedule.step_microbatches(microbatches, target_mbs=target_mbs)
```
Comment (kwen2501) [inline test/test_pipeline_schedule.py:346]: Same here.

**Review by kwen2501 — COMMENTED** [review]
Thanks for enabling the CI. Left one comment.

File: test/test_pipeline_schedule.py (hunk 1 above)
Comment (H-Huang) [reply test/test_pipeline_schedule.py:330]: we don't have to, but if we do that is also allright too since if it is not the last stage then it wont calculate the loss using the target_mbs.

Comment (kwen2501) [reply test/test_pipeline_schedule.py:330]: I'd rather not rely on assumptions :) 
We should test the real case (strictly speaking even `microbatches` should not be fed to every rank).
